src/main/java/org/fusesource/jansi/io/AnsiOutputStream.java

```diff
@@ -112,6 +112,10 @@ public int getTerminalWidth() {
         return width.getTerminalWidth();
     }
 
+    public AnsiProcessor getProcessor() {
+        return processor;
+    }
+
```

Comment:
This method should not be needed (see below).

Reply:
I could remove that by making the below instead, but that would break multithreading:
```java
public class HtmlAnsiOutputStream extends AnsiOutputStream {

    private static AnsiToHtmlProcessor ansiToHtmlProcessor = null;

...

    public HtmlAnsiOutputStream(OutputStream os) {
        super(...);
        ansiToHtmlProcessor.setHtmlAnsiOutputStream(this);
    }

```